bin/index.js

```diff
@@ -208,25 +208,29 @@ app.get('/bundles/npm/:package/:version/:path(*)', (req, res) =>
                     .send(`<${oldModule.getUri()}> <https://linkedsoftwaredependencies.org/vocabularies/npm#maxSatisfying> <${module.getUri()}/${req.params.path}>.`);
 
             return module.getJson().then(json => {
-                let paths = json['lsd:importPaths'];
-                let contexts = json['lsd:contexts'];
-                if ((!paths && !contexts) || !json.dist || !json.dist.tarball)
-                    return res.sendStatus(404);
-
-                let valid = false;
-                if (paths) {
-                    for (let key in paths) {
-                        if (req.params.path.indexOf(paths[key]) >= 0) {
-                            valid = true;
-                            break;
+                // In case lsd:importPaths is true we always need to download the tarball to check
+                let valid = json['lsd:importPaths'] === true;
```

Comment:
This should actually check `lsd:module`. Semantics of `importPaths` hasn't changed.

Reply:
Got confused with all the lsd!